src/camera/Camera.ts

```diff
@@ -271,11 +271,11 @@ abstract class Camera {
    * <ko>{@link ERROR_CODE NOT_ATTACHED_TO_FLICKING} {@link Camera#init init}이 이전에 호출되지 않은 경우</ko>
    * @return {this}
    */
-  public async lookAt(pos: number): Promise<void> {
+  public lookAt(pos: number): void {
     const prevPos = this._position;
 
     this._position = pos;
-    await this._refreshVisiblePanels();
+    this._refreshVisiblePanels();
```

Comment:
The logic seems to be different from the previous one. I wonder if that was part of the intention. The point where EVENTS.VISIBLE_CHANGE occurs after flicking.renderer.render() is finished is the same as before. However, the three methods below can be executed before VISIBLE_CHANGE occurs differently from the existing ones. I wonder if there are any problems with this part.
```js
this._checkNeedPanel();
this._checkReachEnd(prevPos, pos);
this._applyTransform();
```

Reply:
Yes, the timing changed and even the interface of the internal component is changed.
But as there wasn't any previous definition of the timing of `visibleChange`, I hope that's okay.